examples/in-composition/composition.yaml

```diff
@@ -184,7 +184,7 @@ spec:
               string:
                 fmt: "%s-argocd"
     - base:
-        apiVersion: kubernetes.crossplane.io/v1alpha1
+        apiVersion: kubernetes.crossplane.io/v1beta1
```

Comment:
think this needs to be fixed in master branch 

`kubernetes.crossplane.io/v1alpha2`

Reply:
This one is actually the ProviderConfig, we didnt touch versioning of that. But the below one for the Object is wrong, ill fix it. Thanks for noticing!